src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs

```diff
@@ -392,21 +404,23 @@ public async Task<IActionResult> DeleteFile(Guid Id, int sequenceNo, Guid sectio
         }
 
         [HttpPost("/Application/Submit")]
-        public async Task<IActionResult> Submit(Guid applicationId, int sequenceNo)
+        public async Task<IActionResult> Submit(Guid Id)
         {
-            var application = await _applicationApiClient.GetApplication(applicationId);
-            var canUpdate = await CanUpdateApplication(application.ApplicationId, sequenceNo);
+            var userId = await GetUserId();
+            var email = await GetUserEmail();
+            var application = await _applicationApiClient.GetApplication(Id);
+            var activeSequence = await _qnaApiClient.GetApplicationActiveSequence(application.ApplicationId);
+            var canUpdate = CanUpdateApplication(activeSequence);
             if (!canUpdate)
             {
-                return RedirectToAction("Sequence", new { applicationId });
+                return RedirectToAction("Sequence", new { Id });
```

Comment:
Is there a better description of Id - what kind of Id is it?

Reply:
This is an application Id but Qna also has an applicationId to remove confusion used Id for local application class